pyros_schemas/ros/schemagic.py

```diff
@@ -80,28 +81,24 @@ def create(ros_msg_class,
                     ros_schema_inst = RosNested(create(stype))  # we need to nest the next (Ros)Schema
 
         members.setdefault(s, ros_schema_inst)
+        schema_instance.declared_fields[s] = ros_schema_inst
```

Comment:
This seems very hacky to me.
If marshmallow is updated and decides to change the internals of RosSchema, nothing will work and it will be hard to find this place again. Why not initializing the instance after we have gathered all members ?